src/main/java/org/sonar/plugins/stash/StashIssueReportingPostJob.java

```diff
@@ -101,18 +107,7 @@ private void updateStashWithSonarInfo(StashClient stashClient, StashCredentials
         stashRequestFacade.addPullRequestReviewer(pr, stashCredentials.getLogin(), stashClient);
       }
 
-      // build code coverage if requested
-      CoverageIssuesReport coverageReport = new CoverageIssuesReport();
-
-      String codeCoverageSeverity = stashRequestFacade.getCodeCoverageSeverity();
-
-      if (! StringUtils.equals(codeCoverageSeverity, StashPlugin.SEVERITY_NONE)) {
-        coverageReport = stashRequestFacade.getCoverageReport(project.getKey(), context,
-                                                                   inputFileCacheSensor,
-                                                                   codeCoverageSeverity, sonarqubeClient);
-      }
-
-      postInfoAndPRsActions(pr, issueReport, issueThreshold, diffReport, coverageReport, stashClient, sonarqubeClient);
+      postInfoAndPRsActions(pr, issueReport, issueThreshold, diffReport, stashClient);
 
     } catch (StashConfigurationException e) {
       LOGGER.error("Unable to push SonarQube report to Stash: {}", e.getMessage());

```

Comment:
LOGGER.error("Unable to push SonarQube report to Stash: {}", e.getMessage(), e);

or

LOGGER.error("Unable to push SonarQube report to Stash.", e);